### PR TITLE
Fixup eslint violations

### PR DESCRIPTION
**Pull request by hunterwerlla**
- We have a lot of eslint rules disabled, enable a few of them and fix violations

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/awsexplorer/defaultRegion.ts [hunk 1]
```diff
@@ -113,31 +113,31 @@ export async function checkExplorerForDefaultRegion(
 
     const regionHiddenResponse = response[0].label
 
-    switch (regionHiddenResponse) {
```
Comment (justinmk3) [inline src/awsexplorer/defaultRegion.ts:116]: why disallow switch-case?

**Review by justinmk3 — APPROVED** [review]
nice changes

File: src/awsexplorer/defaultRegion.ts (hunk 1 above)
Comment (hunterwerlla) [reply src/awsexplorer/defaultRegion.ts:116]: declaring variables that escape the switch case. In this case though, I thought the logic was very strange, I think it is more clear now.

**Pull request merged**